packages/documentation/src/stories/elements/legend/legend.stories.ts

```diff
@@ -0,0 +1,35 @@
+import type { StoryObj } from '@storybook/web-components';
+import { html } from 'lit';
+import { MetaExtended } from '@root/types';
+
+const meta: MetaExtended = {
+  id: '61faccd2-de2c-48f0-9a06-c051a56580ef',
+  title: 'Typography/Legend',
```

Comment:
This needs some input from the other devs (see https://github.com/swisspost/design-system/issues/3640).

```suggestion
  title: 'Foundations/Typography/Legend',
```